### PR TITLE
Courses: "Find local classes" banner

### PR DESCRIPTION
**Pull request by breville**
Also introduces a simple GridContainer React component which splits the 970px width into a certain number of columns and displays children divs left-to-right (or right-to-left) inside it.

screenshot 2017-07-08 08 13 45

### CODE REVIEW

**Comment by breville** [comment]
cc @tanyaparker

File: apps/src/templates/studioHomepages/GridContainer.jsx [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+  },
+  item: {
+    float: "left",
+  },
+  itemRtl: {
+    float: "right"
+  }
+};
+
+const GridContainer= React.createClass({
```
Comment (Erin007) [inline apps/src/templates/studioHomepages/GridContainer.jsx]: Super handy, versatile new component!

File: apps/src/templates/studioHomepages/FindLocalClassBanner.jsx [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+};
+
+const FindLocalClassBanner = React.createClass({
+  propTypes: {
+    codeOrgUrlPrefix: React.PropTypes.string.isRequired,
+    isRtl: React.PropTypes.bool.isRequired
+  },
+
+  render() {
+    const { codeOrgUrlPrefix, isRtl }= this.props;
```
Comment (Bjvanminnen) [inline apps/src/templates/studioHomepages/FindLocalClassBanner.jsx]: nit: space between } and =

File: apps/src/templates/studioHomepages/GridContainer.jsx (hunk 1 above)
Comment (Bjvanminnen) [reply apps/src/templates/studioHomepages/GridContainer.jsx]: nit: space before =

**Comment by tanyaparker** [comment]
Can you keep "View my projects" button and "Your section" component below Local Classes? Otherwise it'll get lost between HOC and Local Classes.

**Comment by tanyaparker** [comment]
LGTM

**Pull request merged**